setup.py

```diff
@@ -50,6 +50,7 @@ def run(self):
     "colorama>=0.3.9",
     "configobj>=5.0.6",
     "gitpython>3",
+    "dulwich>=0.20.6",
```

Comment:
The only problem is that both dulwich and pygit don't have wheels for 3.9, but that is solvable.